pyston/serializer.py

```diff
@@ -354,6 +344,8 @@ def _get_verbose_value(self, raw, field_or_method, obj, **kwargs):
             return formats.localize(timezone.template_localtime(raw))
         elif isinstance(raw, (datetime.date, datetime.time)):
             return formats.localize(raw)
+        elif raw is None:
+            return '--'
```

Comment:
tohle je trošku čistka, Verbose value pro null jsou dvě čárky. Vypadá to v tabulce mnohem lépe

Reply:
nehodíš to do konstanty?

Reply:
dám to do configu